src/applications/mhv-secure-messaging/tests/e2e/pages/PilotEnvPage.js

```diff
@@ -0,0 +1,56 @@
+import mockPilotMessages from '../fixtures/pilot-responses/inbox-threads-OH-response.json';
+import mockFolders from '../fixtures/pilot-responses/folders-respose.json';
+import { Paths } from '../utils/constants';
+
+class PilotEnvPage {
+  loadInboxMessages = (
+    num = 1,
```

Comment:
not sure if this argument is needed as this always will be 1 in PilotEnvPage class, isn't it?